boltdesignsystem/bolt
# DS-590 new icons for Pega sites

## Jira

https://pegadigitalit.atlassian.net/browse/DS-590

## Summary

New icons added to represent the network of Pega sites.

## Details

SVG files are added for the new icons.

## How to test

Run the branch locally and check out the icon element docs. Make sure all new icons are rendering without any kinks.

## Release notes

### Visual changes

`customer-service` icon has been updated to match brand icon. The double chat bubbles are slightly different from before.
